spec.bs

```diff
@@ -178,6 +178,19 @@ executing the issuance and redemption operations. Issuers make these available
 through secure HTTP [[!RFC8446]] endpoints. User agents should fetch the key
 commitments periodically.
 
+A <dfn>pst key</dfn> is a [=map=] representing an individual cryptographic key with the following entries:
+1. The key "expiry" with a value [=moment=] representing the key's expiration.
+1. Other cryptographic parameters keyed by string keys.
```

Comment:
So, I appreciate the thorough definition here, but I actually think the definition you already have below works well and maybe you could just keep the definition on line 191 (or move it inline to line 175). Something like

```
A <dfn>key commitment</dfn> is a [=map=] representing a collection of cryptographic keys and associated metadata necessary for executing the issuance and redemption operations.
```

in line 175.